client-app/xapi/graphql/account/index.ts

```diff
@@ -9,6 +9,7 @@ export { default as updateMemberAddresses } from "./mutations/updateMemberAddres
 export { default as updatePersonalData } from "./mutations/updatePersonalData";
 export { default as createUser } from "./mutations/createUser";
 export { default as createContact } from "./mutations/createContact";
+export { default as resetPasswordByToken } from "./mutations/resetPasswordByToken";
```

Comment:
add `requestPasswordReset` here

Reply:
it's already added. It is query but not mutation.